react/nasa-picture-of-the-day/app/blogs/[slug]/page.js

```diff
@@ -0,0 +1,13 @@
+import { usePathname } from "next/navigation";
+
+export default function BlogPost() {
+  const pathname = usePathname();
+  const title = pathname.split("/").pop().replace(/-/g, " ");
+}
+
+return (
+  <div>
+    <h1>{title.charAt(0).toUpperCase() + title.slice(1)}</h1>
+    <p>This is the content of {title}.</p>
+  </div>
+);
```

Comment:
This component, unfortunatelly, doesn't work for 2 reasons:
- you have a typo
- component must be a Client Component (used with `'use client'`), because you use hooks there (everything that starts with `use` is a hook)